.travis.yml

```diff
@@ -10,6 +10,7 @@ php:
     - 7.2
     - 7.3
     - 7.4
+    - 8.0snapshot
```

Comment:
Php-cs-fixer does not suppor PHP8 yet, but you can run composer with `--ignore-platform-req=php`

And add this env var:
```
PHP_CS_FIXER_IGNORE_ENV: 1
```

Reply:
Removed the dep for now, as suggested by @Nyholm.